<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>-2</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Saphira&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>63</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>78</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log :</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>73</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>-51</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icon.qrc">:/icon/icon/800x600_Vertikal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPort_open"/>
   <addaction name="actionStart_monitoring"/>
   <addaction name="actionPort_set"/>
  </widget>
  <action name="actionPort_set">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/icon/icon/setting.png</normaloff>:/icon/icon/setting.png</iconset>
   </property>
   <property name="text">
    <string>port_set</string>
   </property>
   <property name="toolTip">
    <string>Setting Port</string>
   </property>
  </action>
  <action name="actionPort_open">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/icon/icon/port.png</normaloff>:/icon/icon/port.png</iconset>
   </property>
   <property name="toolTip">
    <string>Open Port</string>
   </property>
  </action>
  <action name="actionChannel_set">
   <property name="text">
    <string>channel_set</string>
   </property>
  </action>
  <action name="actionChannel_reset">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/icon/icon/reset.png</normaloff>:/icon/icon/reset.png</iconset>
   </property>
   <property name="text">
    <string>channel_reset</string>
   </property>
  </action>
  <action name="actionStart_monitoring">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/icon/icon/play.png</normaloff>:/icon/icon/play.png</iconset>
   </property>
   <property name="text">
    <string>start_monitoring</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
